Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/InsetsHelper.java

```diff
@@ -0,0 +1,182 @@
+package org.dolphinemu.dolphinemu.utils;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.util.DisplayMetrics;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.FrameLayout;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.graphics.Insets;
+import androidx.core.view.ViewCompat;
+import androidx.core.view.WindowInsetsCompat;
+import androidx.core.widget.NestedScrollView;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.slidingpanelayout.widget.SlidingPaneLayout;
+import androidx.viewpager.widget.ViewPager;
+
+import com.google.android.material.appbar.AppBarLayout;
+import com.google.android.material.color.MaterialColors;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import org.dolphinemu.dolphinemu.R;
+
+public class InsetsHelper
+{
+  public static final int FAB_INSET = 16;
+  public static final int EXTRA_NAV_INSET = 32;
```

Comment:
What's the purpose of this constant? If it's intended to be the exact same height as the bar at the bottom of the screen with three navigation buttons, hardcoding it like this seems sketchy considering that devices other than phones usually have a bigger bar...

Reply:
Hm, no, I suppose it's not that since not all the activities are using it. I'm not quite sure what it is, though.